js/api.md

```diff
@@ -362,6 +362,28 @@ Amplify.configure({
 });
 ```
 
+#### AWS AppSync Multi-Auth
+
+AWS AppSync can support multiple authentication modes on a single API. [LINK TO APPSYNC DOCS](http://docs). In order to use this feature with the Amplify Graphql Client the `API.graphql({...})` function would accept an optional parameter called `authMode`, its value will be one of the supported auth modes:
```

Comment:
Multi-auth is for _authorization_ not _authentication_ on a single API. Please update.

Reply:
done